docs/baking.md

```diff
@@ -0,0 +1,159 @@
+<!--
+   - SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+# Baking with tezos-packaging on Ubuntu
+
+Tezos-packaging provides an easy way to install and set up infrastructure for
+interacting with Tezos blockchain. This article provides a step-by-step guide for
+setting up baking instance for Tezos on Ubuntu.
+
+## Prerequisites
+
+### Installing required packages
+
+In order to run baking instance, you'll need the following Tezos binaries:
+`tezos-client`, `tezos-node`, `tezos-baker-<proto>`, `tezos-endorser-<proto>`.
+
+To install them run the following commands:
+```
+# Add PPA with Tezos binaries
+sudo add-apt-repository ppa:serokell/tezos
+sudo apt-get update
+# Install packages
+sudo apt-get install tezos-client tezos-node tezos-baker-<proto> tezos-endorser-<proto>
+```
+
+Packages for `tezos-node`, `tezos-baker-<proto>` and `tezos-endorser-<proto>` provide
+systemd units for running the corresponding binaries in the background
+
+### Downloading zcash-params
+
+Since v8.0 it's required to have `sapling-output.params` and `sapling-spend.params` files from
+the `zcash-params` within the `XDG_DATA_DIRS` to run the `tezos-node`.
+We'll need these params to be accessed by different
+users (because systemd services are usually run as a separate user), thus the suggested way of getting
+them via `https://raw.githubusercontent.com/zcash/zcash/master/zcutil/fetch-params.sh` script which will
+download them to the `HOME` directory of the current user is not convenient. Moreover, it will
+download large `sprout-groth16.params` which is unused at the runtime.
+So, it's better to download them manually to the `/usr/local/share/zcash-params`, in order
+to do that run the following commands:
+```
+sudo mkdir -p /usr/local/share/zcash-params
+sudo wget https://gitlab.com/tezos/opam-repository/-/raw/v8.1/zcash-params/sapling-output.params -O /usr/local/share/zcash-params/sapling-output.params
+sudo wget https://gitlab.com/tezos/opam-repository/-/raw/v8.1/zcash-params/sapling-spend.params -O /usr/local/share/zcash-params/sapling-spend.params
+```
```

Comment:
What do you think about making our own script just for this and putting it into `tezos-packaging/scripts/` instead?

It could be a copy of the one mentioned above (but with a different destination and without `groth16`) or perhaps a simpler new one.

Reply:
Apparently, it makes some sense